ooni/ui/web/client/index.html

```diff
@@ -13,5 +13,5 @@
     <app>
       Loading...
     </app>
-  </body>
+  <script type="text/javascript" src="app.bundle.js?f06164bd3b339e781c75"></script></body>
```

Comment:
Actually that is something generated automatically by webpack when it builds a new version of the app.bundle.js file. The reason why it appends to the app.bundle.js src file a random query string is to bust the cache of browsers that otherwise would not be serving you the newly built file.